Lib/asyncio/locks.py

```diff
@@ -78,12 +78,14 @@ def __init__(self, *, loop=None):
         self._waiters = None
         self._locked = False
         if loop is None:
-            self._loop = events.get_event_loop()
+            self._loop = events._get_running_loop()
+            if self._loop is None:
+                warnings.warn("The creation of asyncio objects outside of a running "
+                              "event loop is deprecated as of Python 3.9.",
+                              DeprecationWarning, stacklevel=2)
+                self._loop = events.get_event_loop()
         else:
             self._loop = loop
-            warnings.warn("The loop argument is deprecated since Python 3.8, "
-                          "and scheduled for removal in Python 3.10.",
-                          DeprecationWarning, stacklevel=2)
```

Comment:
It would be very confusing to users to add a deprecation warning in 3.8, and then remove it from certain asyncio objects in 3.9. That wouldn't make much sense. 

(also applies to the other removals)